datascience/tables.py

```diff
@@ -5678,12 +5678,46 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable view over the rows in a table.
+
+        args:
+        table- accepts a table object of class Table()
+        
+        >>> from datascience import Table
+        >>> t = Table().with_columns({
+             'letter': ['a', 'b', 'c', 'z'],
+             'count':  [  9,   3,   3,   1],
+             'points': [  1,   2,   2,  10],
+        })
+
+        >>> r = Table.Rows(t)
+            print(r)
+
+        Rows(letter | count | points
+        a      | 9     | 1
+        b      | 3     | 2
+        c      | 3     | 2
+        z      | 1     | 10)
+        """
         def __init__(self, table):
             self._table = table
             self._labels = None
 
         def __getitem__(self, i):
+            """
+            Access the i-th row of the given table.
+
+            args:
+            i- index of the Row that needs to be accessed.
+
+            returns:
+            Returns a Row object containing the i-th row of the given table
+
+            >>> r.__getitem__(0)
```

Comment:
To show `__getitem__`, you can just use the `[]` notation. Example: `r[0]` should give you the same result. Change required because it is more Pythonic and easier to read/remember for beginning learners.